src/components/pages/ParentContact/ParentContactContainer.js

```diff
@@ -0,0 +1,122 @@
+    <>
+      <Layout className="parent-dashboard">
+        <ParentNavTopBar />
+        <div className='"site-card-border-less-wrapper"'>
+          <Row className="card">
+            <Col span={16}>
+              <Row className="title">
+                <h4 style={{ color: '#878D93', fontWeight: '700' }}>
+                  CONTACT US
+                </h4>
+              </Row>
+              <div className="description">
+                <Row>
+                  <p>
+                    If you would like to let us know how we're doing of if you
+                    are experiencing issues, contact us by filling in the form
+                    bellow.
+                  </p>
+                </Row>
+              </div>
+            </Col>
+            <Col
+              span={8}
+              type="flex"
+              style={{ alignItems: 'center' }}
+              justify="bottom"
+            >
+              <img
+                style={{ position: 'absolute', overflow: 'scroll' }}
+                alt="example"
+                src={img2}
+              />
+            </Col>
+          </Row>
+        </div>
+        <div className="form-main">
+          <Row className="card">
+            <form className="contact-form-container" onSubmit={handleSubmit}>
+              <Row className="form-inputs">
+                <Col>
+                  <label for="name">Name</label>
+                  <br />
+                  <input
+                    type="text"
+                    name="name"
+                    id="name"
+                    className="contact-form-input"
+                    value={name}
+                    onChange={e => setName(e.target.value)}
+                  />
+                </Col>
+                <Col>
+                  <label for="email">Email Address</label>
+                  <br />
+                  <input
+                    type="email"
+                    name="email"
+                    id="email"
+                    className="contact-form-input"
+                    value={email}
+                    onChange={e => setEmail(e.target.value)}
+                  />
+                </Col>
+              </Row>
+              <Row>
+                <label for="message">Message</label>
+                <br />
+                <textarea
+                  id="message"
+                  name="message"
+                  className="contact-form-textarea"
+                  placeholder="Write your message here"
+                  value={message}
+                  onChange={e => setMessage(e.target.value)}
+                />
+              </Row>
+              <Row>
+                <input
+                  type="submit"
+                  value="SEND MESSAGE"
+                  className="contact-form-submit-btn"
+                />
+              </Row>
+            </form>
+          </Row>
+        </div>
+      </Layout>
+    </>
+  );
+};
+
+export default ParentContactContainer;
```

Comment:
What was the reasoning behind keeping the image below inside the parentContact folder as opposed to in the public folder with the rest of the images?

Reply:
if you do end up moving it over there you might want to give it a more descriptive name then 'img2'